README.md

```diff
@@ -61,15 +61,14 @@ import superjson from 'superjson';
 
 const jsonString = superjson.stringify({ date: new Date(0) });
 
-// jsonString === '{"json":{"date":"1970-01-01T00:00:00.000Z"},"meta":{"values":{date:"Date"}}}'
-```
-
-And parse your JSON like so:
+// jsonString === '{"json":{"date":"1970-01-01T00:00:00.000Z"},"meta":{"values":{date:["Date"]}}}'
 
-```js
+// 
+// And parse your JSON like so:
+// 
 const object = superjson.parse<{ date: Date }>(jsonString);
 
-// object === { date: new Date(0) }
+// object === { date: "1970-01-01T00:00:00.000Z" }
```

Comment:
This will confuse people. The reason we had `new Date(0)` there, was that it conveyed how the date really is a `Date`, and not a `string`. The suggested way of writing doesn't convey that.

Reply:
Please revert.